src/jit/codegenarm64.cpp

```diff
@@ -862,18 +874,90 @@ void                CodeGen::genFuncletProlog(BasicBlock* block)
     if (isFilter)
     {
         // This is the first block of a filter
+        // Note that register x1 = CallerSP of the containing function
+        // X1 is overwritten by the first Load (new callerSP)
+        // X2 is scratch when we have a large constant offset
```

Comment:
Bruce,  I broke any lines that were too long  (greater than 120 characters)
I thought that the coding conventions also stated that lines shouldn't exceed that length.